src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,91 @@
-export const Pagination = () => {};
+import { getNumbers } from "../../utils";
+import cn from "classnames";
+
+type Props = {
+  currentPage: number;
+  items: string[];
+  perPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  currentPage,
+  items,
+  perPage,
+  onPageChange,
+}) => {
+  const handlePrevPage = () => {
+    if (currentPage > 1) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const handleNextPage = () => {
+    if (currentPage < Math.ceil(items.length / perPage)) {
+      onPageChange(currentPage + 1);
+    }
```

Comment:
Similarly, in 'handleNextPage', you should also prevent the default anchor behavior for consistency and to avoid potential issues with the URL changing unexpectedly.